### PR TITLE
CB-2933 - fixed NAMENODE port and JOBTRACKER url

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2 [hunk 1]
```diff
@@ -196,7 +196,7 @@
     <service>
         <role>JOBTRACKER</role>
         {% for hostloc in salt['pillar.get']('gateway:location')['RESOURCEMANAGER'] -%}
-        <url>rpc://{{ hostloc }}:{{ ports['JOBTRACKER'] }}/ws</url>
```
Comment (keyki) [inline orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2]: Why is this removed?

Comment (attilapalfi92) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2]: I'm not sure what's needed and what is not here anymore. @ckoncz-hwx can you elaborate? Shall I leave this here?

Comment (ckoncz-hwx) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2]: The Knox documentation for Oozie service configuration does not say anything about a path in the rpc:// url:
https://knox.apache.org/books/knox-1-3-0/user-guide.html#Oozie
I did run Oozie tests through Knox and they worked fine with a JOBTRACKER url without a `/ws` path.

Comment (keyki) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2]: Right, this is RPC. Thanks.
